solutions/openacc/axpy/axpy_openacc.F90

```diff
@@ -28,6 +28,29 @@ subroutine axpy_gpu(n, alpha, x, y)
 
 end subroutine axpy_gpu
 
+subroutine print_versions()
+
+    integer :: omp_get_num_threads
+!$      character (len=3) :: ompnumthreads0=""
+!$      integer :: cscs_omp_v=0, ompnumthreads1
+!$      cscs_omp_v = _OPENMP
+!$      call getenv("OMP_NUM_THREADS", ompnumthreads0)
+!$omp parallel private(ompnumthreads1)
+!$      ompnumthreads1 = omp_get_num_threads()
+!$omp   master
+!$      print *, "OPENMP version:", cscs_omp_v, &
+!$            ompnumthreads0, ompnumthreads1
+!$omp   end master
+!$omp end parallel
+
+#ifdef _OPENACC
+    print *, "OPENACC version:", _OPENACC
+#endif
+
+! TODO: call MPI_Get_version(version, subversion, iErr)
```

Comment:
Also why you capitalised the extension of this file? I prefer the lowercase. If we decide to go uppercase, we should keep it across all Fortran files.